src/main/java/org/apache/maven/archiver/MavenArchiveConfiguration.java

```diff
@@ -35,6 +35,10 @@ public class MavenArchiveConfiguration {
 
     private boolean recompressAddedZips = true;
 
+    /**
+     * @deprecated See <a href="https://bugs.openjdk.org/browse/JDK-8305597">JDK-8305597</a>
+     */
+    @Deprecated
```

Comment:
no real need to deprecate a private field

Reply:
It's an internal mark that the field should no longer be used, it helps to detect the places where it's used and is a reminder that in the future it could be removed.

Reply:
The point is that the usage of index.list has been marked deprecated and is not being used starting with JDK18 https://www.oracle.com/java/technologies/javase/18-relnote-issues.html

Furthermore as already referenced in JDK21 it will be completely being removed. Also it's good to have a reminder here...

Reply:
Deprecating the method is enough. Private fields can be changed freely without worrying about compatibility when they're no longer used. Adding Javadoc to private fields, deprecation warnings included, is painting the lily.

Reply:
Well, adding Javadoc to private fields could be helpful in some cases, especially if the comment is succinct.

Yet, even if adding a Javadoc on a private field is just a cosmetic change, it doesn't hurt in any way to have it, more when it signals to future developers that touch the code that the field should no longer be used for anything (or with extra care).

I don't mind removing the Javadoc from the field, but there is no real need to remove it, in the end, is just a matter of code convention, just like adding Javadoc to a private method is valid, and that very same class contains Javadoc with an unhelpful `@since` comment.